patoolib/__init__.py

```diff
@@ -814,4 +814,31 @@ def get_inner_file_names_rar(file_path):
     'application/x-rar': get_inner_file_names_rar,
     'application/vnd.rar': get_inner_file_names_rar,
     'application/x-rar-compressed': get_inner_file_names_rar
-}
\ No newline at end of file
+}
+
+def get_inner_file_info(file_path):
+    files_list = []
```

Comment:
Why pass `files_list` into the functions instead of defining the list inside the functions? (it would make sense if both `if` statements could be accessed, and the general list could be modified in both functions, but that's not the case here, right?)